samples/csharp/MicrosoftEduGraphSamples/Workflows/ModuleWorkflow.cs

```diff
@@ -0,0 +1,153 @@
+                Console.WriteLine($"New module has been created: {module.Id} - {module.DisplayName} - {module.Status}");
+                await MicrosoftGraphSDK.Module.SetupResourcesFolder(graphClient, _config["classId"], module.Id);
+
+                // Add a link resource.
+                EducationModuleResource requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationLinkResource
+                    {
+                        OdataType = "#microsoft.graph.educationLinkResource",
+                        DisplayName = "Bing site",
+                        Link = "https://www.bing.com",
+                    },
+                };
+                // Adding a link resource
+                var newResource = await MicrosoftGraphSDK.Module.PostResourceAsync(graphClient, _config["classId"], module.Id.ToString(), requestBody);
+
+                // Add a new word document resource.
+                requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationWordResource
+                    {
+                        OdataType = "#microsoft.graph.educationWordResource",
+                        DisplayName = "test_word_file.docx",
+
+                    },
+                };
+                // A new Word file is added
+                newResource = await MicrosoftGraphSDK.Module.PostResourceAsync(graphClient, _config["classId"], module.Id.ToString(), requestBody);
+
+                // Get the General channel
+                var channels = await MicrosoftGraphSDK.Team.GetChannelsAsync(graphClient, _config["classId"]);
+
+                // Add a channel resource
+                requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationChannelResource
+                    {
+                        OdataType = "#microsoft.graph.educationChannelResource",
+                        Url = "https://graph.microsoft.com/v1.0/teams/"+ _config["classId"] + "/channels/"+ channels.Value[0].Id,
+                        DisplayName = "General",
+                    },
+                };
+                // Added a channel resource
+                newResource = await MicrosoftGraphSDK.Module.PostResourceAsync(graphClient, _config["classId"], module.Id.ToString(), requestBody);
+
+                // Assignment resource
+                var assignment = await MicrosoftGraphSDK.Assignment.CreateAsync(graphClient, _config["classId"]);
+                var assignmentId = assignment.Id;
+                requestBody = new EducationModuleResource
+                {
+                    Resource = new EducationLinkedAssignmentResource
+                    {
+                        OdataType = "#microsoft.graph.educationLinkedAssignmentResource",
+                        Url = "https://graph.microsoft.com/v1.0/education/classes/"+ _config["classId"] +"/assignments/"+assignmentId,
```

Comment:
nit: leave spaces before and after each **+**

Reply:
added space